package.json

```diff
@@ -104,7 +99,21 @@
     "webpack-merge": "^4.2.1",
     "webpack-node-externals": "^1.7.2"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "eslint": "^5.9.0",
+    "eslint-config-smartprix": "^3.5.3",
+    "vue": "^2.5.17",
+    "vue-server-renderer": "^2.5.17",
+    "vue-template-compiler": "^2.5.17"
+
+  },
+  "peerDependencies": {
+    "eslint": "*",
+    "eslint-config-smartprix": "*",
+    "vue": "*",
```

Comment:
Dependencies related to vue are only needed if the project is vue related. `vue` dependencies can be moved to `optionalDependencies`

Reply:
I guess optionalDependencies are installed by default too. Maybe this is the best way.